packages/infra/test/run-cert.ts

```diff
@@ -0,0 +1,16 @@
+import express from "express";
```

Comment:
test script to verify your cert. See README

Reply:
We store these on `packages/utils`, could create a folder for carequality under `./src`

Reply:
Also, please add a header to the file and instructions how to use it. Can refer to other files on `packages/utils`

Reply:
Also, maybe this could be a more descriptive name for the file: `cq-cert-checker.ts` (or similar)